### ISSUE TITLE
Bitcoin needs to be supported in Roboto

### ISSUE THREAD

**Issue by roozbehp** [google/roboto#133]
The proposal is at http://www.unicode.org/L2/L2015/15229-bitcoin-sign.pdf

**Comment by mruddy** [comment]
I agree -- Unicode character U+20BF BITCOIN SIGN was [accepted](https://www.unicode.org/L2/L2015/15254.htm#145-C7) and included in https://www.unicode.org/versions/Unicode10.0.0/ that was released in June 2017.

If it helps, here's a SVG containing the main glyph in the bitcoin repo: https://github.com/bitcoin/bitcoin/blob/master/src/qt/res/src/bitcoin.svg
